### ISSUE TITLE
Using ECMWF pressure profile to adjust moleculare absorption

### ISSUE THREAD

**Issue by BaptisteVandecrux** [BaptisteVandecrux/pySICE#30]
From the ATBD:

The molecular optical thickness can be approximated as (Hansen and Travis, 1974; Iqbal, 1984):

τ_mol (λ)=qλ^(-υ),                                              (A.23)

where the wavelength is in microns. The parameters (q, υ) are subject to variations depending on the atmospheric state. We use as default values the following parameters at the normal pressure and temperature:  q=0.008735,  υ=4.08 (Hansen and Travis,1974). We derive the value of molecular optical thickness at another pressure level p using the following expression: 

τ_mol (λ)=  p^ x τ_m(λ),

where  p ̂=p/p_0,  p is the site pressure, p_0=1013.25 mb. 

The site pressure is calculated using the following equation: p=p_0  exp⁡(-z/H).  Here  z is the height of the underlying surface provided in OLCI files. It is assumed that the scale height H is equal to 6km. **The correct value of p ̂ can be derived from the information on the pressure profile at a given location ( e.g., from the ECMWF re-analysis).** 
>

**Comment by wehrad** [comment]
This is a similar idea than the assimilation of the CAMS AOD 550 for aerosols. From what I've seen so far, it looks like the spatial resolution is way lower than our 1km/500m target, which might unfortunately create some annoying artifacts... Or at least, make the results harder to interpret.  What do you think?

**Comment by BaptisteVandecrux** [comment]
You are right that both CAMS AOD and ECMWF datasets are at a coarser resolution. 
However they woud replace very rough approximations: spatio-temporally constant aerosol AOD and simple elevation-dependent parametrization for the atmospheric pressure.

The Pressure should be easy to extract from the the tie point information distributed with the OLCI data.

The CAMS dataset is from a complete different source and might be more complicated to incroporate in NRT production.
We agreed during the last PM meeting that using an aerosol climatology (see #31) was a good first step and easy to implement everywhere in the Arctic.

All of this needs testing and has likely a marginal impact on the retrieval.

**Comment by wehrad** [comment]
Thanks for the update! In the recording of the PM only Jason's mic is audible, so I missed quite a lot!

**Comment by BaptisteVandecrux** [comment]
If implementing the climatology lead to visible changes in retrievals, then we will need to go forward to the CAMS product.
